app/config/sculpin_site.yml

```diff
@@ -335,3 +335,7 @@ contributors:
     github: //github.com/edwardangert
     avatar: //avatars2.githubusercontent.com/u/17991901
     linkedin: //www.linkedin.com/in/edward-angert/
+  ccjjmartin:
```

Comment:
@ccjjmartin feel free to add a bio line here if you'd like, either in this PR or a new one if this is already merged when you see this.

Reply:
P.S. Or any of the other available fields, of course!